### PR TITLE
Add callbacks example

### PR DESCRIPTION
**Pull request by Cawllec**
## Additions
- Adds an example of how to add callbacks to Symfony
- Demonstrates how to achieve result of discussion in [#410](https://github.com/bugsnag/bugsnag-php/pull/410)

## TODO:
- Update documentation across examples

### CODE REVIEW

File: example/symfony31/app/AppKernel.php [hunk 1]
```diff
@@ -29,6 +29,19 @@ public function registerBundles()
         return $bundles;
     }
 
+    public function boot()
+    {
+        parent.boot();
```
Comment (GrahamCampbell) [inline example/symfony31/app/AppKernel.php]: While this is technically valid syntax, it does not do what you think! This takes the value of the constant `parent`, and attempts string concatenation with the value obtained from evaluating the global function `boot`.

Comment (kattrali) [reply example/symfony31/app/AppKernel.php]: Huh, thanks @GrahamCampbell - looks like this should've been `parent::boot()`

**Comment by Cawllec** [comment]
Updated examples docs

File: example/symfony31/app/AppKernel.php (hunk 1 above)
Comment (GrahamCampbell) [reply example/symfony31/app/AppKernel.php]: Yeh. GitHub's syntax highlighter had trouble believing it wasn't a method invocation too. :)

**Pull request merged**